8Knot/assets/color.css

```diff
@@ -72,3 +72,8 @@ blockquote p {
 .alert-warning {
     background-color: #B5B682 !important;
 }
+
+/* updates background of the dmc-multiselect in visualizations */
+.viz-values {
```

Comment:
I'd rename this to something descriptive like "dmc-multiselect-custom" because CSS is such a pain to interpret later out of context. Otherwise this works for me.